app/Nova/Resources/Integration.php

```diff
@@ -67,29 +68,37 @@ public function fields(NovaRequest $request): array
 
         return [
             ID::make()
-                ->readonly(),
+                ->readonly()
+                ->onlyOnDetail(),
+
+            DateTime::make('Date created', 'created_at')
+                ->readonly()
+                ->exceptOnForms()
```

Comment:
Do we also show it inside the detail view? There it can be found inside the Activity Log.